snapshot-labs/sx-monorepo
# fix: fix character count on textarea not updating

### Summary



Closes: #395

### How to test

1. Go to any textarea with a character limit
2. FIll it over the limit
3. The character count should reflect the lengh of the textarea (before, always 0)
4. Large number should be pretty formatted (e.g `2,500`)

Will need for https://github.com/snapshot-labs/sx-monorepo/pull/417 to be merged first
